documentation/docs/30-add-ons/05-drizzle.md

```diff
@@ -0,0 +1,51 @@
+---
+title: drizzle
+---
+
+## Usage
+
+```bash
+npx sv add drizzle
+```
+
+## Description
+
+[Drizzle ORM](https://orm.drizzle.team/) is a headless TypeScript ORM with a head.
```

Comment:
headless with a head? lol. I just checked and that's their tagline, but it's completely unintelligible, so I think we should write our own description for it :smile: